website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,128 @@
+* Removing the nodes when the nodes are no longer needed
+
+In many cases, you can configure a simple Karpenter provisioner when it is first installed and not change it again. 
+Other times, you might want to continue to tweak the provisioner or even create multiple provisioners for a cluster where you tailor the requirements of different teams to use specific instance types, zones, or computer architectures.
+Likewise, you might want to instruct the provisioner to take advantage of underlying cloud features, such as AWS spot instances.
+
+As someone deploying pod workloads, you can make specific requests for capacity and features you want from the nodes running your pods.
+If the Kubernetes scheduler is unable to meet those requirements, Karpenter is designed to quickly create the best possible nodes to meet those needs.
+
+## Karpenter concepts
+
+The following are some key concepts you should understand about Karpenter.
+
+### Karpenter provisioners
+
+A Karpenter provisioner controller runs on a Kubernetes cluster.
+Its job is to add nodes to handle unschedulable pods, schedule pods on those nodes, and remove the nodes when they are not needed.
+The controller's work is configured through [Provisioner CRDs](/docs/provisioner-crd/).
+
+Rather than take over the process of provisioning nodes and scheduling pods, the provisioner controller integrates with the Kubernetes scheduler, cluster autoscaler, and related Kubernetes features to make sure that pods are properly and efficiently deployed.
+It also works through Kubernetes to request needed assets from the cloud provider.
+
+Here are some attributes of the Karpenter provisioner:
+
+* **Unschedulable pods**: Karpenter only looks at pods that the Kubernetes scheduler identifies as unschedulable.
+If the Kubernetes scheduler is able to schedule a pod, the Karpenter provisioner takes no action on it.
+* **Provisioner CRD**: Provisioner CRDs define how the provisioner controller behaves.
+Each CRD contains constraints that impact the nodes that can be provisioned and attributes of those nodes (such timers for removing nodes).
+See [Provisioner CRD](/docs/provisioner-crd/) for a description of settings.
+* **Well-known labels**: The CRD can use well-known Kubernetes labels to allow pods to request only certain instance types, computer architectures, operating systems, or other attributes when creating nodes.
+* **Time to live**: A provisioner can also include time-to-live values to indicate when nodes should be decommissioned after a set amount of time from when it was created or after it becomes empty of deployed pods.
+* **Multiple provisioner CRDs**: An important feature of Karpenter is that it allows multiple provisioner CRDs on the same cluster.
+One possible result is that you could configure different teams on the same cluster to run on completely separate capacity.
+One team could run on GPU nodes while another uses storage nodes, with each having a different set of defaults.
```

Comment:
It's important to note that this can be handled by one provisioner as well. The reason you might want two is if you want to isolate the nodes (e.g. billing) or if you want different defaults (i.e. no gpus for this team) or behaviors (i.e. expiration/scale down).